src/lib/core/fiber.c

```diff
@@ -557,6 +557,15 @@ fiber_set_joinable(struct fiber *fiber, bool yesno)
 		fiber->flags &= ~FIBER_IS_JOINABLE;
 }
 
+void
+fiber_set_system(struct fiber *fiber, bool yesno)
+{
+	if (yesno == true)
```

Comment:
Checkpatch doesn't like that. But this is done for consistency with all other `fiber_set_*`. Possibly it's needed to be patched